types/transliteration/index.d.ts

```diff
@@ -0,0 +1,70 @@
+// Type definitions for transliteration 1.6
+// Project: https://github.com/andyhu/transliteration#readme
+// Definitions by: Anthony Trinh <https://github.com/tony19>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare module 'transliteration' {
```

Comment:
**Note:** I needed to declare the module (and disable a couple lint rules) because I needed to declare a top-level namespace. I needed the namespace for `transliterate.config()` and `slugify.config()`, but let me know if there's a better/proper way.